cpan/perlfaq/lib/perlfaq.pod

```diff
@@ -1380,7 +1380,7 @@ patches and corrections over the years.
 =head1 AUTHOR AND COPYRIGHT
 
 Tom Christiansen wrote the original version of this document.
-brian d foy C<< <bdfoy@cpan.org> >> wrote this version. See the
+brian d foy C<< <briandfoy@pobox.com> >> wrote this version. See the
```

Comment:
perlfaq is maintained elsewhere. See https://github.com/perl-doc-cats/perlfaq.

Reply:
sure, but they refuse to accept my contributions.

Reply:
They took mine: https://github.com/perl-doc-cats/perlfaq/commit/b07aaf2b6de891e173213fafe223033aacc1a0d0

Reply:
> but they refuse to accept my contributions

Speaking as one of the perlfaq people, I'm not refusing any contributions. At the time of that particular PR, I was interpreting it as still open for discussion and not yet ready to merge.  If that has changed, could you please add a comment to clarify?

Reply:
I don't know what discussion needs to happen to fix links: https://github.com/perl-doc-cats/perlfaq/pull/111

But, that repo is so toxic I'm just avoiding it. I did leave the patches you can manually apply though.